cfgov/v1/models/learn_page.py

```diff
@@ -316,20 +322,6 @@ class EventPage(AbstractFilterPage):
             ],
             heading="Venue Address",
         ),
-        MultiFieldPanel(
-            [
-                FieldPanel("venue_image_type"),
-                FieldPanel("venue_image"),
-            ],
-            heading="Venue Image",
-        ),
-        MultiFieldPanel(
-            [
-                FieldPanel("post_event_image_type"),
-                FieldPanel("post_event_image"),
-            ],
-            heading="Post-event Image",
```

Comment:
We should remove these fields from the model and not just from the admin panels, if they're not used, on lines 247-267 above here. A quick query suggests there's one `EventPage` with `post_event_image_type` of `"image"`.